test/controllers/ReviewDetailsControllerSpec.scala

```diff
@@ -242,50 +238,6 @@ class ReviewDetailsControllerSpec extends PlaySpec with OneServerPerSuite with M
             }
           }
 
-          "return agent registration page correctly for Agents" in {
```

Comment:
Are all of these tests to be removed ? Should they not be updated to use mockNewAgentRegistrationService instead?